backend/tests/codes/test_views.py

```diff
@@ -84,34 +84,34 @@ def test_get_codes_list(client):
     number_of_example_codes = 4
     list_of_codes = factories.CodeFactory.create_batch(number_of_example_codes, is_example=True)
 
-    url = reverse('code-list')
+    url = reverse('codes-list')
     response = client.get(url)
 
     assert response.status_code == status.HTTP_200_OK
     assert len(response.data) == number_of_example_codes
 
 
 @pytest.mark.django_db
-def test_get_codes_list_no_examples(client):
+def test_get_codes_list_no_examples_filtering_by_example_true(client):
     number_of_example_codes = 4
     list_of_codes = factories.CodeFactory.create_batch(number_of_example_codes, is_example=False)
 
-    url = reverse('code-list')
-    response = client.get(url)
+    url = reverse('codes-list')
+    response = client.get(url + "?example=True")
```

Comment:
Nie musisz łączyć stringi, automatycznie robi to client :D 
https://stackoverflow.com/a/23451655/13787426

Reply:
Właśnie nie mogłem tego znaleźć, szukałem po docsach i ani jednego przykładu z tym XD

Reply:
Dzięki 🚀